src/content/docs/ko/reference/errors/no-prerendered-routes-with-domains.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: Prerendered routes aren't supported when internationalization domains are enabled.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+import DontEditWarning from '~/components/DontEditWarning.astro'
+
```

Comment:
It is missing this [part](https://github.com/withastro/docs/pull/6727/files#diff-3065da4cdaa30776bdde3ac80f4d513e404a4f13651a537750fb1d15afc60063R16)